docs/user/docs/tokens.md

```diff
@@ -0,0 +1,55 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Token Authentication
+
+To authenticate against the API you can either use a Bearer token (which the
+browser usually uses) or authenticate with a longer lived basic authentication
+token which can be used e.g. in scripts.
+
+## Token Creation
+
+To create a Token you can go to Profile > Token and insert a short description
+to create a token.
+
+ <!-- prettier-ignore -->
+
+!!! info The password which is generated can only be copied. Make sure you save
+it - you won't be able to access it again
```

Comment:
This is broken: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/831d68f6-0551-4cd0-846f-51f7ebafbb25)